catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1192,7 +1192,7 @@ public QueryResponse query(QueryRequest queryRequest, FederationStrategy strateg
                     LOGGER.throwing(federationException);
                     throw federationException;
                 } else {
-                    LOGGER.debug("FederationStratgy was not specified, using default strategy: "
+                    LOGGER.debug("FederationStrategy was not specified, using default strategy: "
```

Comment:
If you are going to fix spelling here, then might as well change from + operator to {} substitution